### PR TITLE
Fix Negative value display Z-offset and babystep

### PR DESCRIPTION
**Pull request by billydragon**
### Requirements

* Filling out this template is required. Pull Requests without a clear description may be closed at the maintainers' discretion.

### CODE REVIEW

File: TFT/src/User/Configuration.h [hunk 1]
```diff
@@ -56,20 +56,20 @@
...
 
 // Size of machine
-#define X_MIN_POS 0
+#define X_MIN_POS -10
 #define Y_MIN_POS 0
 #define Z_MIN_POS 0
-#define X_MAX_POS 235
-#define Y_MAX_POS 235
-#define Z_MAX_POS 250
+#define X_MAX_POS 410     // Anycubic Chiron
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]: Please try not to push settings only specific to your printer,these may create problem to others who do not know much about compiling the firmware, try to keep the defaults,

File: TFT/src/User/Configuration.h [hunk 2]
```diff
@@ -56,20 +56,20 @@
...
+#define Y_MAX_POS 410     // Anycubic Chiron
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]:
```suggestion
#define Y_MAX_POS 235
```

File: TFT/src/User/Configuration.h [hunk 3]
```diff
@@ -56,20 +56,20 @@
...
+#define Z_MAX_POS 450     // Anycubic Chiron
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]:
```suggestion
#define Z_MAX_POS 250
```

File: TFT/src/User/Configuration.h [hunk 4]
```diff
@@ -56,20 +56,20 @@
 #define SPEED_ID         {"Sp.", "Fr."}
 
 // Default move speed mm/min
-#define DEFAULT_SPEED_MOVE      3000
+#define DEFAULT_SPEED_MOVE      3600
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]:
```suggestion
#define DEFAULT_SPEED_MOVE      3000
```

File: TFT/src/User/Configuration.h [hunk 5]
```diff
@@ -16,13 +16,13 @@
 // Marlin Mode Background & Font Color Options
 // Current color options from lcd.h: BLACK, BLUE, BROWN, BRRED, CYAN, GBLUE, GRAY, GREEN, MAGENTA, RED, WHITE, YELLOW
 #define ST7920_BKCOLOR BLACK
-#define ST7920_FNCOLOR GREEN
+#define ST7920_FNCOLOR GRAY
 
 // Text displayed at the top of the LCD in Marlin Mode.
-//#define ST7920_BANNER_TEXT "LCD12864 Simulator"
+#define ST7920_BANNER_TEXT "Anycubic Chiron - SKR V1.3"
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]:
```suggestion
#define ST7920_BANNER_TEXT "LCD12864 Simulator"
```

File: TFT/src/User/Configuration.h [hunk 6]
```diff
@@ -16,13 +16,13 @@
...
-#define ST7920_FNCOLOR GREEN
+#define ST7920_FNCOLOR GRAY
 
 // Text displayed at the top of the LCD in Marlin Mode.
-//#define ST7920_BANNER_TEXT "LCD12864 Simulator"
+#define ST7920_BANNER_TEXT "Anycubic Chiron - SKR V1.3"
 
 // Run Marlin Mode fullscreen. Not recommended for TFT24.
-//#define ST7920_FULLSCREEN
+#define ST7920_FULLSCREEN
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]:
```suggestion
//#define ST7920_FULLSCREEN
```

File: TFT/src/User/Configuration.h [hunk 7]
```diff
@@ -56,20 +56,20 @@
...
+#define DEFAULT_SPEED_MOVE      3600
 
 // Extrude speed mm/min
 #define EXTRUDE_SLOW_SPEED      60
 #define EXTRUDE_NORMAL_SPEED    600
 #define EXTRUDE_FAST_SPEED      1200
 
 // Size of machine
-#define X_MIN_POS 0
+#define X_MIN_POS -10
```
Comment (guruathwal) [inline TFT/src/User/Configuration.h]: may cause problem to other type of printer
```suggestion
#define X_MIN_POS 0
```

**Comment by extesy** [comment]
@billydragon please revert all changes specific to Anycubic Chiron

**Comment by guruathwal** [comment]
@billydragon you tried to remove the changes but they are added again with you commit.
You need to make a copy of source folder and change setting for your printer in that one.

**Comment by bigtreetech** [comment]
merged in #219

**Pull request closed without merging**